exporter/prometheusremotewriteexporter/exporter.go

```diff
@@ -25,37 +25,41 @@ import (
 	"math"
 	"net/http"
 	"net/url"
+	"strings"
 	"sync"
 
 	"github.com/gogo/protobuf/proto"
 	"github.com/golang/snappy"
 	"github.com/prometheus/prometheus/prompb"
 
+	"go.opentelemetry.io/collector/component"
 	"go.opentelemetry.io/collector/consumer/consumererror"
 	"go.opentelemetry.io/collector/consumer/pdata"
 	"go.opentelemetry.io/collector/internal"
 	otlp "go.opentelemetry.io/collector/internal/data/protogen/metrics/v1"
-	"go.opentelemetry.io/collector/internal/version"
 )
 
 const (
 	maxConcurrentRequests = 5
 	maxBatchByteSize      = 3000000
+	prwVersion            = "0.1.0"
 )
 
 // PrwExporter converts OTLP metrics to Prometheus remote write TimeSeries and sends them to a remote endpoint.
 type PrwExporter struct {
-	namespace      string
-	externalLabels map[string]string
-	endpointURL    *url.URL
-	client         *http.Client
-	wg             *sync.WaitGroup
-	closeChan      chan struct{}
+	namespace       string
+	externalLabels  map[string]string
+	endpointURL     *url.URL
+	client          *http.Client
+	wg              *sync.WaitGroup
+	closeChan       chan struct{}
+	startInfo       component.ApplicationStartInfo
```

Comment:
Is anything using `startInfo` after it is used in `NewPrwExporter`?  If not, you can probably remove it from this struct and just store the UA string.